packages/adapter-drizzle-all/src/pg/schema.ts

```diff
@@ -0,0 +1,48 @@
+import { integer, timestamp, pgTable, text, primaryKey } from 'drizzle-orm/pg-core';
+import { drizzle } from 'drizzle-orm/postgres-js';
```

Comment:
There are serverless drivers as well that needs to be taken into consideration. Ex: I use `drizzle-orm/neon-serverless` driver for postgres instances hosted by neon. More drivers here https://github.com/drizzle-team/drizzle-orm/blob/main/drizzle-orm/src/pg-core/README.md

Reply:
I'm under the impression that the drivers are handled in userland code.

Users will pass their final database client to `next-auth` which will use whichever SQL flavor the user's driver is using. As long as your SQL flavor matches the `next-auth` adapter you're using, you should be good to go.

That's at least my impression of things; hope I'm not being a fool. 😀